rosidl_typesupport_connext_cpp/include/rosidl_typesupport_connext_cpp/impl/rosidl_generator_c/message_type_support.h

```diff
@@ -0,0 +1,45 @@
+// Copyright 2016 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This header contains the connext specific macros used to generate the
+// name of the function used to retrieve the type support for a message.
+
+// TODO(wjwwood): move this into it's own C specific type support package
```

Comment:
So, this is going to happen in the next set of changes, then (creating rosidl_typesupport_connext_c)?